test/network_autoyast_test.rb

```diff
@@ -319,4 +319,33 @@ def keep_install_network_value(value)
       end
     end
   end
+
+  describe "#valid_rename_udev_rule?" do
+    it "fails when the rule do not contain new name" do
+      rule = { "rule" => "exists", "value" => "as_well" }
```

Comment:
Please use meaningful values when it does not hinder the test. Copying the values from the bug should do it.
>        <rule>
>          <name>eth0</name>
>          <rule>ATTR{address}</rule>
>          <value>00:02:29:69:d3:63</value>
>        </rule>